include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -438,7 +438,7 @@ struct __sycl_usm_alloc
 
 // impl for USM pointer
 template <typename _ExecutionPolicy, typename _T, typename _BValueT>
-struct __buffer<_ExecutionPolicy, _T, _BValueT*>
```

Comment:
@MikeDvorskiy could you please clarify is this type of buffer (based on USM memory) really used in code?
Where?

Reply:
As I can see the specialization `__buffer_impl<_ExecutionPolicy, _T, _BValueT*>` is not used in OneDPL and should be removed.

Reply:
@MikeDvorskiy, thank you for this comment.
I am going to delete this `__buffer_impl<_ExecutionPolicy, _T, _BValueT*>` in the https://github.com/oneapi-src/oneDPL/pull/1218